30__testing_software/20__check.robot

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Suite Setup       Suite Setup
+Suite Teardown    Suite Teardown
+Library           OperatingSystem
+Library           ../SailfishSDK.py
+Resource          ../SailfishSDK.robot
+
+*** Test Cases ***
+Basic Check
```

Comment:
The way "check" is used is always the same. The difference is in the app. So maybe "Basic Packaging" if you don't like just "Basic"?